ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts

```diff
@@ -121,6 +135,7 @@ export const useNewMhrRegistration = () => {
   }
 
   return {
-    buildApiData
+    buildApiData,
+    staffPayment: parseStaffPayment
```

Comment:
This is a little confusing to me, is this the property or the function expression? 
The type indicates the expression, so why not return `parseStaffPayment` ?

Reply:
We are returning a function, that will be called later. Just used it for naming. Will update.